soc/espressif/esp32s3/Kconfig.amp

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2023 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_ESP32S3
+
+menu "AMP config"
+
+config ESP32S3_APPCPU_IRAM_SIZE
+	hex "ESP32S3 APPCPU IRAM size"
+	depends on SOC_ESP32S3_PROCPU || SOC_ESP32S3_APPCPU
+	default 0x10000
+	help
+	  Defines APPCPU IRAM area size in bytes.
+
+config ESP32S3_APPCPU_DRAM_SIZE
+	hex "ESP32S3 APPCPU DRAM size"
+	depends on SOC_ESP32S3_PROCPU || SOC_ESP32S3_APPCPU
+	default 0x10000
+	help
+	  Defines APPCPU DRAM area size in bytes.
+
+config SOC_ENABLE_APPCPU
+	bool "Utilize CPU1 of ESP32S3"
```

Comment:
If this config is hidden you can remove the option prompt, otherwise it will became user selectable.